docs/Advanced-Usage/Workloads/Defining-Custom-Workloads.rst

```diff
@@ -1,208 +0,0 @@
-.. _defining-custom-workloads:
```

Comment:
I think we should keep this file around for now, because it documents what the manager expects to ingest for it's workload json configuration files (unless this is moved somewhere else?)

Reply:
word. I didn't look at it carefully enough before. I've moved it to under "Manager Usage" to reflect that it's specific to the firesim command. The old link redirects you there now. I also updated it slightly, in particular, I added an "attention" warning that it is largely deprecated in favor of the FireMarshal 'install' command. Obviously this stuff will need to be unified at some point, but this should be good enough for now.